src/Umbraco.Core/Models/PublishNotificationSaveOptions.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Umbraco.Cms.Core.Models;
+
+public enum PublishNotificationSaveOptions
```

Comment:
You could make this a `[Flags]` enum.  Then when you are using it, you could just check using `.HasFlag()` for `Saving` or `Saved` rather than having to check the `All` option too.

For reference we use this in Deploy:
https://github.com/umbraco/Umbraco-Deploy/blob/09bd48ecb9a83990692e027d9d1e9d661b0ec261/src/Umbraco.Deploy.Core/Configuration/DeployConfiguration/DeploySettings.Content.cs#L42

Like this:
https://github.com/umbraco/Umbraco-Deploy/blob/09bd48ecb9a83990692e027d9d1e9d661b0ec261/src/Umbraco.Deploy.Infrastructure/Connectors/ServiceConnectors/ContentConnector/DocumentConnector.cs#L855

Would suggest adding an XML header comment just explaining what this is used for too.

Reply:
Done 😁